src/pages/LP/Header/Header.jsx

```diff
@@ -8,31 +8,35 @@ import logo from '../../../images/lpLogo.png'
 
 
 
-export default function Header() {
-    const [queueCount, setQueueCount] = useState(0)
-
+export default function Header({socket}) {
     const { isAuthenticated } = useContext(AuthContext);
-    const { request } = useHttp();
-    const navigate = useNavigate();
+    const navigate = useNavigate(); 
+    const [queueCount, setQueueCount] = useState(null);
 
     useEffect(() => {
-        async function requestQueueCount() {
-            const data = await request(`${process.env.REACT_APP_API_URL}/api/landing/queuecount`, 'POST')
-            setQueueCount(data.count)
-        }
-        requestQueueCount();
-    },[])
-
-    function handleClick(e) { //delete andsunctionality
-        console.log('User authorized: ', isAuthenticated)
-        isAuthenticated ? navigate('/profile') : navigate('/signin')
-    }
+        socket?.emit('getQueueCount');
+        socket?.on('getQueueCount', (data) => {
+            console.log('[SocketIO] -> Emited from server (getQueueCount)', data);
+            setQueueCount(data);
+        })
+
+        socket?.on('changeQueueCount', (data) => {
+            console.log('[SocketIO] -> Emited from server (changeQueueCount)', data);
+            setQueueCount(data);
+        })
+    }, [socket]) 
 
     return (
         <HeaderContainer>
             <Image src={logo}/>
-            <h1> Current queue count: {queueCount} </h1>
-            <HeaderButton onClick={ e => handleClick(e) } type="button" value={ isAuthenticated ? "MY QUEUES" : "SIGN IN" } />
+            <h1> {queueCount ? `Current queue count: ${queueCount}` : `Not connected`} </h1>
+           
+
+            <HeaderButton 
+                onClick={ e => {isAuthenticated ? navigate('/profile') : navigate('/signin')} } 
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/OrangeMGE-ITP/SmartQ-ITP-Front/pullRequest?prid=8748018)
```suggestion
                onClick={ (e) => {isAuthenticated ? navigate('/profile') : navigate('/signin')} } 
```